### PR TITLE
Put and Post Request with Json data directly

### PR DESCRIPTION
**Pull request by tvainika**
Put Request and Post Request allow optional json parameter which will be json encoded data similar to files. This just wraps functionality of requests library, and works similar to recently added json parameter of Get Request

### CODE REVIEW

**Comment by vkosuri** [comment]
@tvainika thank for the PR, the PR failing due to lack of client certificates updation, this will be resolved by this PR https://github.com/bulkan/robotframework-requests/pull/171

**Comment by vkosuri** [comment]
@tvainika I have restarted this job, hope this will pass with latest changes in master.

**Comment by mederic-p** [comment]
Hey, will this get merged soon?

**Comment by bulkan** [comment]
I just checked [requests](https://github.com/requests/requests/blob/master/tox.ini) still supports python 2.6 so we should too.

What do we need to do to get the tests running in py2.6 ?

**Comment by tvainika** [comment]
> I just checked requests still supports python 2.6 so we should too.
> 
> What do we need to do to get the tests running in py2.6 ?

`master` branch does **not** run tests against py2.6 anymore (https://github.com/bulkan/robotframework-requests/blob/master/.travis.yml). Therefore merging this PR should result green travis.

**Comment by mederic-p** [comment]
Hey, can we get an updated version on pypi with these changes?

File: src/RequestsLibrary/RequestsKeywords.py [hunk 1]
```diff
@@ -413,6 +413,7 @@ def post_request(
             alias,
             uri,
             data=None,
+            json=None,
```
Comment (kormbrek-rally) [inline src/RequestsLibrary/RequestsKeywords.py]: Late to bring this up @bulkan, but inserting new arguments like this is not backwards compatible. If someone was calling `Post Request` with positional arguments, the value for `params` became the value for `json`. Adding `json` to the end would have avoided this. This is a great feature to have btw. When going from Python usage of requests, have been missing convenience of `json` arg.

**Pull request merged**